_data/beyond-sidebar.yml

```diff
@@ -488,6 +488,12 @@
             - title: Mark order as viewed
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-mark-order-as-viewed
               id: mark_order_as_viewed
+            - title: Mark multiple orders as viewed
+              link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-order-batch-actions-mark-as-viewed
+              id: mark_multiple_orders_as_viewed
+            - title: Mark multiple orders as unviewed
```

Comment:
This one raises the question why there is no "Mark order as unviewed" endpoint, but maybe this is just not needed at the moment :D

Reply:
We don't have this endpoint because there is no API client that needs it.